Gems/Multiplayer/Code/Include/Multiplayer/MultiplayerEditorServerBus.h

```diff
@@ -24,4 +24,38 @@ namespace Multiplayer
         virtual void SendEditorServerLevelDataPacket(AzNetworking::IConnection* connection) = 0;
     };
     using MultiplayerEditorServerRequestBus = AZ::EBus<MultiplayerEditorServerRequests>;
+
+    class MultiplayerEditorServerNotifications : public AZ::EBusTraits
+    {
+    public:
+        static const AZ::EBusHandlerPolicy HandlerPolicy = AZ::EBusHandlerPolicy::Multiple;
+        static const AZ::EBusAddressPolicy AddressPolicy = AZ::EBusAddressPolicy::Single;
+
+        //! Notification when the Editor has successfully opened the editor server.
+        virtual void OnServerLaunched() {}
+
+        //! Notification when the Editor has failed to open the editor server.
+        //! Might have failed for various reasons, this is a catch-all.
+        virtual void OnServerLaunchFail() {}
+        
+        //! Notification when the Editor attempts a tpc connection to the server.
+        //! Note: It's possible multiple connection attempts are needed if the user starts and stops editor playmode a lot of times because the server port take a few seconds to be free.
```

Comment:
```suggestion
        //! Note: It's possible multiple connection attempts are needed if the user starts and stops editor play mode repeatedly because the server port takes a few seconds to free.
```

Reply:
fixed!